index.html

```diff
@@ -132,191 +145,214 @@ <h3>Looking for inspirations (Random recipes)</h3>
       </div>
     </section>
 
-    <!-- NUTRITIONAL ANALYSIS SECTION -->
-    <section class="nutritional-analysis-section">
+    <!-- *** NUTRITIONAL ANALYSIS SECTION *** -->
+    <section
+      class="section container-fluid"
```

Comment:
Add class: d-none

Reply:
@willmowlam doesn't the script.js add the d-none class to the section on load already?